practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/model/PractitionerIdProviderLink.java

```diff
@@ -0,0 +1,28 @@
+package uk.ac.ox.ndph.mts.practitioner_service.model;
+
+public class PractitionerIdProviderLink {
```

Comment:
Going with the naming convention you set already - shouldn't this be something like UserAccount?